sbt-scalajs-cross/src/main/scala/scalajscross/ScalaJsCross.scala

```diff
@@ -0,0 +1,41 @@
+package scalajscross
+
+import sbtcross._
+
+import sbt._
+
+import org.scalajs.sbtplugin.{ScalaJSPlugin, ScalaJSCrossVersion}
+
+case object JSPlatform extends Platform {
+  def identifier: String = "js"
+  def enable(project: Project): Project =
+    project.enablePlugins(ScalaJSPlugin).settings(crossPlatform := JSPlatform)
```

Comment:
As I said elsewhere, the `.settings(crossPlatform := JSPlatform)` part must not be in `enable`. It must be automatically there even with only `enablePlugins(ScalaJSPlugin)`.

Obviously you cannot add it directly in `ScalaJSPlugin` for now, as you have no control over it. But you can declare a separate `ScalaJSCrossPlugin`, which is an `AutoPlugin` automatically triggered by `ScalaJSPlugin`. That auto-plugin can specify `crossPlatform := JSPlatform`.